src/main/scala/org/constellation/consensus/RoundManager.scala

```diff
@@ -196,24 +195,25 @@ class RoundManager(config: Config)(implicit dao: DAO) extends Actor with ActorLo
 
   private[consensus] def resolveMissingParents(
     roundData: RoundData
-  )(implicit dao: DAO): Future[List[Option[CheckpointCache]]] = {
+  )(implicit dao: DAO): Future[List[CheckpointCache]] = {
 
     implicit val ec = dao.edgeExecutionContext
 
-    val cbToResolve = roundData.tipsSOE
+    roundData.tipsSOE
       .filterNot(t => dao.checkpointService.contains(t.baseHash).unsafeRunSync())
-      .map(_.baseHash)
+      .map(_.baseHash) match {
+      case Nil => Future.successful(List.empty)
```

Comment:
do you have auto-format on save enabled? wrong indent :P

Reply:
bump

Reply:
I think it was before I have enabled it, will fix though

Reply:
It's funny after running reformat I still have same indents.